### PR TITLE
#2089 - Remove 'Edit poll' menu

### LINKED ISSUE #2088: Easy: remove the ability to edit a poll
### Problem

Screenshot 2024-06-18 at 9 17 25 AM

Clicking edit results in:

Screenshot 2024-06-18 at 9 17 33 AM

This is unprofessional, and it really doesn't make sense to edit a poll anyway (except maybe before anyone has voted, but in that case deleting the poll and reposting is just as good).

### Solution

Remove the edit button on polls.

### LINKED ISSUE #2089: Bug: can't upload multiple files on mobile
### Problem

On Android, when I go to pick a picture using the file picker, I can only choose one file at a time, and then when I choose another file it replaces the first picture that I chose.

This means there's no way to post multiple pictures at once on mobile (at least on Android, I haven't tested iOS).

### Solution

Ideally, you should be able to pick multiple files at once in the media picker.

Alternatively, if that is impossible for some reason, then you should be able to add files to the existing set of files that are being attached.

### PR DESCRIPTION
**Pull request by SebinSong**
closes #2088

### CODE REVIEW

File: frontend/controller/actions/identity.js [hunk 1]
```diff
@@ -798,7 +798,7 @@ export default (sbp('sbp/selectors/register', {
     }
 
     if (deleteResult?.some(r => r.status === 'rejected')) {
-      console.error('[gi.actions/identity/removeFiles] Some CIDs could not be deleted', deleteResult.map((r, i) => r.status === 'rejected' && toDelete[i]).filter(Boolean))
+      console.error('[gi.actions/identity/removeFiles] Some CIDs could not be deleted', deleteResult?.map((r, i) => r.status === 'rejected' && toDelete[i]).filter(Boolean))
```
Comment (SebinSong) [inline frontend/controller/actions/identity.js]: This change is unrelated to the issue itself but fixing the Flow linter error found in `master`

**Review by taoeffect — CHANGES_REQUESTED** [review]
Nicely done @SebinSong! And especially thanks for fixing the `flow` error in `master` that somehow snuck through!

I found one issue, and that's that now we can no longer delete polls after they're opened:

Screenshot 2024-06-19 at 10 34 44 AM

**Comment by SebinSong** [comment]
@taoeffect fixed that issue.

File: frontend/views/containers/chatroom/MessageActions.vue [hunk 2]
```diff
@@ -164,10 +164,11 @@ export default ({
       return this.isText || this.isPoll
     },
     isEditable () {
-      return this.isMsgSender && (this.isText || this.isPoll)
+      return this.isMsgSender && this.isText
     },
     isDeletable () {
-      return this.isEditable || this.isGroupCreator
+      return this.isGroupCreator ||
+        (this.isMsgSender && (this.isText || this.isPoll))
```
Comment (taoeffect) [inline frontend/views/containers/chatroom/MessageActions.vue:171]: Is there a situation where we wouldn't want a message to be deletable when `this.isMsgSender` is true?

Comment (SebinSong) [reply frontend/views/containers/chatroom/MessageActions.vue:171]: @taoeffect Yes there is. This kind of notification message is one of that example.

image

**Review by taoeffect — APPROVED** [review]
Great work @SebinSong!

**Pull request merged**